mala/common/physical_data.py

```diff
@@ -116,11 +123,20 @@ def read_from_openpmd_file(self, path, units=None):
         data = np.zeros((mesh["0"].shape[0], mesh["0"].shape[1],
```

Comment:
Note: Zeroing memory is much more costly than just allocating it would be bit less memory-safe, but more efficient to just use `np.empty` here.